packages/cli/src/commands/admin-web/local-dev.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2020 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { serveBundle } from '../../lib/bundler';
+import path from 'path';
+import { ConfigReader } from '@backstage/config';
+
+console.log('CLI spoolin');
+
+async function main() {
+  /* eslint-disable-next-line no-restricted-syntax */
+  const targetDir = path.resolve(__dirname, 'embedded/');
+  const waitForExit = await serveBundle({
+    entry: path.resolve(targetDir, 'index'),
+    targetDir,
+    checksEnabled: false,
+    frontendAppConfigs: [],
+    frontendConfig: new ConfigReader(
+      {
+        app: {
+          baseUrl: 'http://127.0.0.1:7008',
```

Comment:
nit: we can just run this on `3000` right? Either way port 7008 is a bit weird out of place for a web uit? Or maybe I am looking to deep into this.

Reply:
`3000` is cool, but Backstage (I guess backend, heh) runs on `7007`, and I figured that we _probably_ don't want to conflict, "just in case"? I'm not attached to `7008` though :)